cucumber/api/features/authenticate.feature

```diff
@@ -21,6 +21,58 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
+  Scenario: Authenticate response should be encoded if Accept-Encoding equals base64
+    When I authenticate Alice with Accept-Encoding header "base64"
+    Then the HTTP response content type is "text/plain"
+    And the HTTP response is base64 encoded
+    And user "alice" has been authorized by Conjur
+    And there is an audit record matching:
+    """
+      <86>1 * * conjur * authn
+      [auth@43868 authenticator="authn"]
+      [subject@43868 role="cucumber:user:alice"]
+      [action@43868 operation="authenticate" result="success"]
+      cucumber:user:alice successfully authenticated with authenticator authn
+    """
+  Scenario: Authenticate response should be encoded if Accept-Encoding includes base64
+    When I authenticate Alice with Accept-Encoding header "base64,gzip,defalte,br"
+    Then the HTTP response content type is "text/plain"
+    And the HTTP response is base64 encoded
+    And user "alice" has been authorized by Conjur
+    And there is an audit record matching:
+    """
+      <86>1 * * conjur * authn
+      [auth@43868 authenticator="authn"]
+      [subject@43868 role="cucumber:user:alice"]
+      [action@43868 operation="authenticate" result="success"]
+      cucumber:user:alice successfully authenticated with authenticator authn
+    """
+  Scenario: Authenticate response should be encoded if Accept-Encoding includes base64 with mixed case and spaces
+    When I authenticate Alice with Accept-Encoding header "gzip      ,  bASe64 ,defalte,br"
+    Then the HTTP response content type is "text/plain"
+    And the HTTP response is base64 encoded
+    And user "alice" has been authorized by Conjur
+    And there is an audit record matching:
+    """
+      <86>1 * * conjur * authn
+      [auth@43868 authenticator="authn"]
+      [subject@43868 role="cucumber:user:alice"]
+      [action@43868 operation="authenticate" result="success"]
+      cucumber:user:alice successfully authenticated with authenticator authn
+    """
+
+  Scenario: Authenticate response should be json if Accept-Encoding doesn't include base64
```

Comment:
what about a test where `Accept-Encoding` is not present that validates the content-type is `application/json`?

Reply:
Other test do that also we have a test that sends `Accept-Encoding` without `base64` that verifies `application/json` returned

Reply:
thanks, i thought that you added this step.

Reply:
I did add it.... authenticate.feature:66